tests/cli/test_cli.py

```diff
@@ -21,6 +27,34 @@ def generate_test_input_file():
     return file_path
 
 
+def assert_equal_lists(res, expected):
+    assert len(expected) == len(res)
+    assert all([a == b for a, b in zip(expected, res)])
+
+
+def test_make_bento_name_docker_compatible():
```

Comment:
This is a good use case for [parametrized tests](https://docs.pytest.org/en/stable/parametrize.html#:~:text=%40pytest.,custom%20parametrization%20schemes%20or%20extensions). Internally, `pytest` will run a separate fixture with each set of parameters, so if there's a failure, you'll see exactly which set of parameters failed.